include/qpdf/Pl_Buffer.hh

```diff
@@ -35,7 +35,6 @@
 
 #include <qpdf/Buffer.hh>
 #include <qpdf/Pipeline.hh>
-#include <qpdf/PointerHolder.hh>
```

Comment:
I had left all these `#include <qpdf/PointerHolder.hh>` statements in the public header files so that people who were using `PointerHolder` and getting it indirectly through other things would get the PointerHolder warnings instead of an error about `PointerHolder` not being defined, so in general, I don't want to pull them out until qpdf 12. I should have been explicit about that somewhere. I added a comment to #785 as a reminder.

Reply:
I'm going to merge as is, then restore the include and add comments to all the includes.

Reply:
Noted for the future.